auto_submit/lib/service/validation_service.dart

```diff
@@ -39,37 +35,25 @@ class ValidationService {
     /// Validates a PR marked with the reverts label.
     revertValidation = Revert(config: config);
     approverService = ApproverService(config);
-
-    validations.addAll({
```

Comment:
Yes. I refactored this to the ValidationFilter class. It allows us to manage the validations done on a particular type of pull request rather than putting a bunch of if/else clauses. These are all still there.